src/main/webapp/add-course.html

```diff
@@ -38,9 +38,10 @@
                     </select>
                 </div>
                 <textarea name="school-name" id="school-name" placeholder="School Name" maxlength=1500></textarea>
-                <input type="submit"/>
+                <input type="submit"/ id="add-button">
```

Comment:
super nit: / is in the wrong place

Reply:
I'd say this is more significant than a nit actually, since it's not just stylistic but actually malformed. (The reason this doesn't actually break things is because web browsers have been designed to accept html with mistakes like this.)

Reply:
omg totally didn't notice that - thanks!